frontend/src/utilities/segmentIOUtils.tsx

```diff
@@ -75,7 +75,7 @@ export const initSegment = async (props: {
       const key = analytics.methods[e];
       analytics[key] = analytics.factory(key);
     }
-    analytics.load = (key: string, e: Event) => {
+    analytics.load = (key: string, options: any) => {
```

Comment:
We have a no `any` policy. 

Looks like the type is https://github.com/segmentio/analytics-next/blob/master/packages/browser/src/core/analytics/index.ts#L86 -- maybe we type some of this internally at some point... but I think you just need to stop eslint from having an issue.

```suggestion
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    analytics.load = (key: string, options: any) => {
```